ctapipe/tools/dl1_merge.py

```diff
@@ -0,0 +1,359 @@
+"""
+Merge DL1-files from stage1-process tool
+"""
+from pathlib import Path
+import sys
+import tables
+import os
+from argparse import ArgumentParser
+import numpy as np
+
+from tqdm import tqdm
+
+from ..core import Provenance
+from ctapipe.core import Tool, traits
+from traitlets import List
+from ctapipe.instrument import SubarrayDescription
+
+import warnings
+
+warnings.filterwarnings("ignore", category=tables.NaturalNameWarning)
+
+PROV = Provenance()
+
+required_nodes = [
+    "/dl1/monitoring/subarray/pointing",
+    "/dl1/monitoring/telescope/pointing",
+    "/dl1/service/image_statistics",
+    "/dl1/service/image_statistics.__table_column_meta__",
+    "/dl1/event/subarray/trigger",
+    "/dl1/event/telescope/trigger",
+    "/dl1/event/telescope/parameters",
+    "/dl1/event/telescope/images",
+    "/configuration/simulation/run",
+    "/simulation/event/subarray/shower",
+    "/simulation/event/telescope/parameters",
+    "/simulation/event/telescope/images",
+    "/simulation/service/shower_distribution",
```

Comment:
It might be necessary to not force this. Or mabye add an option `--do-no-require-shower-distribution` since most of our test files are truncated.